components/scream/src/physics/p3/scream_p3_interface.F90

```diff
@@ -180,6 +181,7 @@ subroutine p3_main_f90 (dtime,zi,pmid,dpres,ast,ni_activated,nc_nuceat_tend,q,FQ
 
     integer(kind=c_int) :: it, its, ite, kts, kte
     logical(kind=c_bool) :: do_predict_nc = .true.
+    logical(kind=c_bool) :: do_prescribed_CCN = .true.
```

Comment:
This is kind of a rumination to myself rather than something to change: I was thinking you'd just implement the function interfaces for the C++, but you actually implemented the full capability within these functions. We should make sure to unit test both do_prescribed_CCN=.true. and .false. in this case. Maybe you have - I haven't checked yet. I was also thinking that we can't get nccn_prescribed values in C++ yet since we read them from netcdf... but we do that in p3_interface which we don't run in our P3 unit tests. Instead, we hardcode values to pass to our functions... so we can easily test nccn_prescribed values now.